translation/messages.pot

```diff
@@ -317,971 +326,977 @@ msgstr ""
 
 # Indicator strings:
 
-msgid "RSI"
+msgid "RSI
"
```

Comment:
Why is there `^M` here...? These are Indicator strings and practically doesn't change. If you don't want to debug the `yarn translation` code (I suspect it's a windows issue), just retain the translation strings you want and discard the rest.